### PR TITLE
Add Auditree rules for cloud.gov-hosted apps.

### PR DESCRIPTION
**Pull request by rahearn**
First cut at adding a simple component encapsulating testable best practices for cloud.gov-hosted apps.

Still to do:

- [x] decide if auditree rules should be bundled in this docker image, or delivered separately.
- [x] decide if the docker image is the best delivery for the component, or if it should be hosted within devtools-compliance

Work towards https://github.com/GSA-TTS/devtools-program/issues/90

### CODE REVIEW

**Comment by rahearn** [comment]
Leaning towards delivering auditree in its own docker image.

Leaning towards keeping the component definition here, at least for now.

File: templates/component-definitions/devtools_cloud_gov/component-definition.json [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+            "remarks": "rule_ssh_access_disabled_1"
+          },
+          {
+            "name": "Rule_Id",
+            "value": "ssh-access-disabled",
+            "remarks": "rule_ssh_access_disabled_2"
+          },
+          {
+            "name": "Rule_Description",
+            "value": "Production spaces should disable ssh access",
```
Comment (zjrgov) [inline templates/component-definitions/devtools_cloud_gov/component-definition.json:49]: suggestion (non-blocking): I'm not sure I'm reading this correctly (or that it matters at this stage) but one of these `rule_ssh_access_disabled_2`'s is not like the other.
```suggestion
            "value": "Production orgs should disable ssh access",
```

**Review by zjrgov — APPROVED** [review]
I only skimmed the JSON but the shell stuff LGTM

**Comment by pburkholder** [comment]
Next time there's a Lean Coffee for the Sec & Compliance guild, would you
want to do a little walk through of this?

File: templates/component-definitions/devtools_cloud_gov/component-definition.json (hunk 1 above)
Comment (rahearn) [reply templates/component-definitions/devtools_cloud_gov/component-definition.json:49]: Will update this the next time the file is touched.

**Pull request merged**